connector/queue/model_view.xml

```diff
@@ -140,6 +140,21 @@
             <field name="search_view_id" ref="view_queue_job_search"/>
         </record>
 
+        <record id="action_button_done_queue_job" model="ir.actions.server">
+            <field name="name">Set to 'Done'</field>
+            <field name="model_id" ref="model_queue_job"/>
+            <field name="state">code</field>
+            <field name="code"><![CDATA[
+self.button_done(cr, uid, context['active_ids'], context=context)
```

Comment:
@zakiuu A test into the testsuite would be greatly appreciated.
* set the job state to done,
*  mock the _change_job_sate method (`with mock.patch(Job, '_change_job_state', .... as mocked_change_job:` )
*  call job.button_done()
*  check that the mock has not been called. (`self.assertEqual(0, mocked_change_job.call_count)`)